pkg/build/builder/util_linux.go

```diff
@@ -15,6 +17,10 @@ import (
 	s2iapi "github.com/openshift/source-to-image/pkg/api"
 )
 
+var (
+	selinuxLabelPattern = regexp.MustCompile(`"system_u:object_r:container_file_t:(.*?)"`)
```

Comment:
container_file_t will not work on a RHEL system.  We don't have that type yet.
svirt_sandbox_file_t will work on RHEL and Fedora.

Reply:
Which is going to cause issues across platforms.  I would really prefer we just use the standard tools provided by the selinux bindings.

Reply:
This needs to be fixed before we can do the next round of testing.

Reply:
yes i'm working on adapting to the selinux lib that @rhatdan suggested, but we don't currently vendor it, so it's a bit of work.